.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "➡️ Tracking"
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity and is not currently being tracked. If it still needs to be addressed,
+  please leave a comment and provide additional details, if necessary.
```

Comment:
May I suggest an edit here?

> Is this still causing friction? If so, please comment with any updates or addition details.

I've been experimenting with simpler comments from stale lately and it seems to work well. Leaving out the "we'll close it if you don't" allows people to focus on the goal of solving the friction vs just keeping the issue open.